test/test-cases/typescript-css-modules/app/sku.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  entry: {
+    client: 'src/client.tsx',
+    render: 'src/render.tsx',
+    server: 'src/server.tsx'
```

Comment:
Why do we have all three here? This isn't really something we'd want to recommend.

Reply:
For brevity it serves the test to ensure all paths are covered rather than having to build separate apps/configs in the test.